src/api/passwordApi.js

```diff
@@ -0,0 +1,47 @@
+const checkEmail=(emailId)=>{
+  try {
+    const response = await axios.post(`${apiUrl}api/v1/forgot-password`, {
+      email: emailId,
+    });
+    const resetToken = response.data.data.emailToken;
+    return navigate(`/password/reset/${resetToken}`);
+  } catch (errorResponse) {
+    if (errorResponse.response) {
+      if (errorResponse.response.status === 404) {
+        return setError({ status: true, message: 'Email Not Found' });
+      }
+      return setError({ status: true, message: 'Internal server error' });
+    }
+    if (errorResponse.request) {
+      return setError({ status: true, message: 'Internal server error' });
+    }
+    return setError({ status: true, message: 'Internal server error' });
+  }
```

Comment:
Don't redefine the error message as the backend server will appropriate error message. DRY